src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -142,6 +151,19 @@ module.exports = React.createClass({
             );
         }
 
+        if (this.props.mxEvent.getType() === 'm.room.message') {
+            const content = this.props.mxEvent.getContent();
+            if (content.msgtype // truthy check msgtype
+            && content.msgtype !== 'm.bad.encrypted'
```

Comment:
since the change of the line limit I have changed this to a single line in my working dir